apps/demo/src/app/cart/components/cart-items/cart-items.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="demo-cart-items">
+  <div class="demo-cart-items__content">
+    <demo-cart-item *ngFor="let item of cart.items" [item]="item"></demo-cart-item>
+    <div *ngIf="cart.items.length === 0">
+      There are no items in your cart.
+    </div>
+  </div>
+</div>
```

Comment:
Do you discourage wrapper elements at component root in general? Should I give the cart component the same treatment?

Reply:
Generally I don't usually care too much, though I usually like to get rid of any code that isn't necessary. I'm not sure if there could be a case where the wrapper would be necessary. Here, it just bothered me, because there were two layers of unneeded wrapping elements.